state-of-charge/interface.js

```diff
@@ -0,0 +1,60 @@
+import { getRoute } from './client';
+import { drawRoutePolyline } from './map';
+import { getDurationString } from '../utils';
+
+export const getStateOfCharge = () => {
+  return document.getElementById('range').value;
+};
+
+/**
+ * Function that renders the header details
+ * @param { object } data - All available route data
+ */
+export const renderRouteHeader = data => {
+  const routeDistance = data.distance ? `${(data.distance / 1000).toFixed(0)} km` : 'Unknown';
```

Comment:
This data needs to update when a user drags the slider and alters the route.

Reply:
Fixed